File: Anul 2/Medii de Programare si Proiectare/Lab/mpp-proiect-csharp-Mothyque/ProiectMPP/ProiectMPP.Gui/ViewModels/MessageController.cs
using System.Windows;

namespace ProiectMPP.Client.ViewModels
{
    public abstract class MessageController : Window
    {
        protected void ShowErrorMessage(string message)
        {
            MessageBox.Show(message, "Error", MessageBoxButton.OK, MessageBoxImage.Error);
        }

        protected void ShowInfoMessage(string message)
        {
            MessageBox.Show(message, "Information", MessageBoxButton.OK, MessageBoxImage.Information);
        }
    }
}

File: Anul 2/Medii de Programare si Proiectare/Lab/mpp-proiect-java-Mothyque/backend/Client/src/main/java/org/example/controller/MessageController.java
package org.example.controller;


import javafx.scene.control.Alert;

public abstract class MessageController
{
    protected void showErrorMessage(String message)
    {
        Alert alert = new Alert(Alert.AlertType.ERROR);
        alert.setTitle("Error");
        alert.setContentText(message);
        alert.showAndWait();
    }

    protected void showInfoMessage(String message)
    {
        Alert alert = new Alert(Alert.AlertType.INFORMATION);
        alert.setTitle("Information");
        alert.setContentText(message);
        alert.showAndWait();
    }
}
